curriculum/challenges/english/08-coding-interview-prep/rosetta-code/longest-increasing-subsequence.md

```diff
@@ -0,0 +1,90 @@
+---
+id: 5e6dd139859c290b6ab80292
+title: Longest increasing subsequence
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+The longest increasing subsequence problem is to find a subsequence of a given sequence in which the subsequence's elements are in sorted order, lowest to highest, and in which the subsequence is as long as possible. An example:
+For the following array:
+$\{3, 10, 2, 1, 20\}$
```

Comment:
I think the $s need to be removed and replaced with some other tag?

Reply:
@ojeytonwilliams I have no idea about this. Is there some markdown we can use for this or should I let the $s be?